api.toml

```diff
@@ -45,8 +45,14 @@
         # /address/:address/esdt will return the list of esdt tokens for a given account
         { Name = "/:address/esdt", Open = true },
 
-        # /address/:address/esdt/:tokenName will return data of an esdt token for a given account
-        { Name = "/:address/esdt/:tokenIdentifier", Open = true }
+        # /address/:address/esdt/:tokenIdentifier will return the balance for a given account and esdt tokenID
+        { Name = "/:address/esdt/:tokenIdentifier", Open = true },
+
+        # /address/:address/nft/:tokenIdentifier/nonce/:nonce will return data of an nft esdt token for a given account, tokenID and nonce
+        { Name = "/:address/nft/:tokenIdentifier/nonce/:nonce", Open = true },
+
+        # /address/:address/completeEsdts will return the list of esdt tokens and all the underlying data for a given account
+        { Name = "/:address/completeEsdts", Open = true }
```

Comment:
this one was removed but I see that it remained in the api toml

Reply:
I removed it in the commit with hash `88d91cdb4febcbc34f1092dc299eb66ae7fe3ab6` but the PR of that commit isn't yet merged. So you can leave it as it is for now